content/knowledge-base/load-firebase-configuration.md

```diff
@@ -27,14 +27,8 @@ Instead of committing the Firebase configuration files to your repository, you c
 
   PROJECT_ROOT=$FCI_BUILD_DIR/myproject/path      # ADD YOUR PROJECT FOLDER PATH HERE
 
-  echo $ANDROID_FIREBASE_SECRET | base64 --decode > $PROJECT_ROOT/android/app/google-services.json
-  echo $IOS_FIREBASE_SECRET | base64 --decode > $PROJECT_ROOT/ios/Runner/GoogleService-Info.plist
+  echo $ANDROID_FIREBASE_SECRET > $PROJECT_ROOT/android/app/google-services.json
+  echo $IOS_FIREBASE_SECRET > $PROJECT_ROOT/ios/Runner/GoogleService-Info.plist
   ```
 
-{{<notebox>}}
-
-`post-clone script failed on base64 decode. The command could not be found`
-If you received this error message when using the script above, it might be due to some invisible Unicode characters in the script after copy-pasting it. Try removing the space between the `base64` command and the `--decode` option and then adding it back.
-{{</notebox>}}
-
-For more details about using base64 and loading the Firebase configuration in Codemagic, see the step-by-step guide [here](https://blog.codemagic.io/how-to-load-firebase-config-in-codemagic-with-environment-variables/).
+For more details about loading the Firebase configuration in Codemagic, see the step-by-step guide [here](https://blog.codemagic.io/how-to-load-firebase-config-in-codemagic-with-environment-variables/).
```

Comment:
can you also confirm the reference here is up to date? if it's not, let's remove it

Reply:
All changes done!